docs/oceanic/phraseology.md

```diff
@@ -1,43 +1,41 @@
 ---
-  title: Phraseology
+  title: Procedures
 ---
 
 --8<-- "includes/abbreviations.md"
 
-## Phraseology
-
-### HF Radio
+## HF Radio
 
 HF radio is used as the primary long-range radio communications medium. The nature of HF radio makes it highly vulnerable to atmospheric distortion and noise, and so radiotelephony (R/T) procedures on HF tend to be more formal in order to maximise clarity.
 
 !!! Example
-    **UAL873** -> **ZAK_W_FSS**: "San Francisco Radio, San Francisco Radio, UAL873 on 122.5"  
-    **ZAK_W_FSS** -> **UAL873**: "UAL873, San Francisco Radio, go-ahead"  
```

Comment:
Can these callsigns be altered to reflect the updated procedures in ZAK? They control the whole thing now the majority of the time, logon code ZAK_FSS.

Reply:
Fixed